torch_geometric/nn/models/deep_graph_infomax.py

```diff
@@ -51,10 +51,12 @@ def discriminate(self, z, summary, sigmoid=True):
 
         Args:
             z (Tensor): The latent space.
+            summary (Tensor): The summary vector.
```

Comment:
Not totally sure why this one is here, it also seems to be in master.

Reply:
Looks like something went wrong during rebase.

Reply:
Yeah, I'll rebase and squash my PRs once the rest of the conversations are resolved